### PR TITLE
HTTP response status codes

### PR DESCRIPTION
**Pull request by GiselleLeeOSI**
Draft of HTTP response status codes

### CODE REVIEW

**Comment by GiselleLeeOSI** [comment]
Please ignore the other two files and just review the status codes.

**Comment by GiselleLeeOSI** [comment]
@vtouati1009 , Thank you for reviewing. In which file and line can I find "id"?

**Comment by vtouati1009** [comment]
Hi, Giselle. My apologies, Line 17, 2nd sentence..."By specifying the stream view id in the call..." :-)

File: Documentation/Status_Codes.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+uid: ocsStatusCodes
+---
+
+# HTTP response status codes
+The server issues HTTP response status codes when a client makes a request over HTTP.
+In response to the requests made in OSIsoft Cloud Services, the system returns the code listed in the table below.
```
Comment (lasato-OSI) [inline Documentation/Status_Codes.md:7]: Change: ...returns the code listed ....
To: returns one of the codes listed ....

File: Documentation/Status_Codes.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+## Status codes table
+|   Status Code             |    Description                                                                    |
+|--------------             |------------------------------------------                                         |
+|``200 OK``                     |   The request was successful.                                                     |
+|``201 Created``                |   A resource was created as a result of a successful request.                     |
+|``202 Accepted``               |   The request was accepted and has been queued for processing.                    |
+|``204 No content``             |   The request was successful and the server is not returning any content.         |
+|``207 Multi-status``           |   The request was partially successful.                                           |
+|``302 Found``                  |   The requested resource will be redirected.                                      |
+|``400 Bad request``            |   Due to a client error, the server does not understand the request.              |
```
Comment (lasato-OSI) [inline Documentation/Status_Codes.md:19]: Change: ...the server does not understand the request. 
To: ...the server cannot process this request. 

I know it's a nit. "Understand" implies that the computer has the human capacity to grasp the meaning of the request.

File: Documentation/Status_Codes.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+|``401 Unauthorized``           |   Authentication has failed or has not been provided.                             |
+|``403 Forbidden``              |   Unauthorized. Make sure the user has the right role to access the resource. |
+|``404 Not found``              |   The server could not find the requested resource.                               |
+|``405 Method not allowed``     |   The resource does not allow the method in the request.                          |
```
Comment (lasato-OSI) [inline Documentation/Status_Codes.md:23]: You're repeating what is in the message. I wonder if there's a way to be more specific. "The resource does not support this method." ???

File: Documentation/Status_Codes.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+|``408 Operation timed out``    |   The server timed out while waiting for a client request.                        |
+|``409 Conflict``               |   The request conflicts with the current state of the server.                     |
+|``500 Internal server error``  |   The server ran into an unexpected condition and failed to complete the request. |
+|``503 Service unavailable``    |   The server is not ready to handle the request. Try again later.                 |
```
Comment (lasato-OSI) [inline Documentation/Status_Codes.md:27]: Change:  ... is not ready to handle the request.
To: ... is unable to handle the request.

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 5]
```diff
@@ -11,10 +11,10 @@ Using a stream view to leverage existing SdsType properties is preferable to cre
 
 You can either see the impact of the stream view on a stream in an ad hoc manner through a GET method or assign the stream view to a stream with a PUT method.
 
-To see how SdsStreamView impacts a stream, you can apply a stream view to any read or GET operation. For more information, see [Read data API](xref:sdsReadingDataApi).
+You can view the impact of the stream view on a stream with a GET method or assign the stream view to a stream with a PUT method. For more information, see [Reading with SdsStreamViews](xref:sdsReadingData#reading-with-sdsstreamviews).
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/SDS_Views.md:14]: I would change to something like, "Use a GET or PUT method to assign a stream view to a stream, and display the stream data specified for the selected stream view."

**Comment by GiselleLeeOSI** [comment]
All your comments have been incorporated and are reflected in the latest commit. If it's not too much trouble, please take a look again and explicitly "approve". Once again, you only need to look at one file: Status_Codes.md

**Pull request merged**